CMakeLists.txt

```diff
@@ -73,7 +69,11 @@ macro(targets)
   )
 endmacro()
 
-call_for_each_rmw_implementation(targets GENERATE_DEFAULT)
+# For whatever reason when I enable this along with Connext, it fails to
+# link.  Disable for now.
+#call_for_each_rmw_implementation(targets GENERATE_DEFAULT)
```

Comment:
This could be due to the typesupport refactor that happened a few months ago https://github.com/ros2/ros2/issues/302. did you try to remove the `GENERATE_DEFAULT` option ? I think you can even get rid of the macro (put everything in top level cmake) now that typesupport is decided at runtime